tests/conftest.py

```diff
@@ -0,0 +1,113 @@
+import os
+
+import numpy as np
+import pandas as pd
+import pytest
+
+import audb
+import audeer
+import audformat
+import audiofile
+
+
+pytest.NAME = 'db'
+pytest.VERSION = '1.0.0'
+
+
+@pytest.fixture
+def db(tmpdir, scope='function'):
```

Comment:
I am wondering about the scope of the fixture. Would it not make sense to have (a) a fixture that publishes the databases with scope "session" and (b) to have a derived fixture with scope "function" that actually loads it ?
While functionally it does not matter, does that not create lots of write io in the way it is implemented?

Reply:
Another solution might be to not publish the database to the `tmpdir` provided by `pytest`, but to a global one.

I will look into this as I also think that it is a better solution to publish the database only ones and not for all tests.

Reply:
Could one also pass the `tempdir` fixture? Afaics it is scoped to the session. Then the location of the repo would be known and could be passed as the 'host' variable.

Reply:
I just forgot to pass the `publish_db` fixture to the `db` fixture.
Now everything works as expected and you can use the `db` fixture in tests to get access to a database, which would then be loaded and is only published ones by the `publish_db` fixture.
In addition, you could use `pytest.NAME`, `pytest.VERSION` together with the `publish_db` fixture access the database if you want to load it yourself.

Reply:
I would have also thought that this is not necessary as `publish_db` is an `autouse`. Anyway this looks good imo and I am resolving it.

Reply:
`autouse` works for other functions I define in `tests/test_fixture.py`, but it seems that it does not work when creating other fixtures. But anyway, the good news is that it works ;)